src/BugsnagUnity/Native/Fallback/CacheManager.cs

```diff
@@ -44,7 +44,14 @@ public CacheManager(Configuration configuration)
         {
             _configuration = configuration;
             CheckForDirectoryCreation();
-            RemoveExpiredPayloads();
+            try
+            {
+                RemoveExpiredPayloads();
+            }
+            catch
+            {
+                //not avaliable in unit tests
+            }
```

Comment:
Since `RemoveExpiredPayloads()` catches any exceptions itself, do we need a `try` block here?

Reply:
yes, because even accessing _cachedEvents during a unit test causes the test to fail

Reply:
i should change the way we access _cachedEvents and _cachedSessions to make this cleaner.

Reply:
Done 👍